app/views/application/_flash.html.erb

```diff
@@ -7,3 +7,5 @@
     </div>
   <% end %>
 </div>
+
+<%= svelte_component "Alerts" %>
```

Comment:
What if the `.alert` element in `Alerts.svelte` was factored out into e.g. `Alert.svelte`, and we used `svelte_component "Alert"` to render the alerts from flash?

Reply:
Hmm actually, what you just said made me realise we can render the flash as json as a prop, and pass it to the svelte component.